src/social-card/highlight-card/highlight-card.service.ts

```diff
@@ -40,11 +40,15 @@ export class HighlightCardService {
   ) {}
 
   private async getHighlightData (highlightId: number): Promise<HighlightCardData> {
-    const highlightReq = await firstValueFrom(this.httpService.get<DbUserHighlight>(`https://api.opensauced.pizza/v1/user/highlights/${highlightId}`));
+    const highlightReq = await firstValueFrom(
+      this.httpService.get<DbUserHighlight>(`https://api.opensauced.pizza/v1/user/highlights/${highlightId}`),
```

Comment:
Just an observation, but we should be able to control the API through an environment variable for deployment so everything isn't point to the production API.

Not for this PR though

```suggestion
      this.httpService.get<DbUserHighlight>(`${process.env.API_BASE_URL}/v1/user/highlights/${highlightId}`),
```